src/Symfony/Component/Cache/Traits/RedisCluster6ProxyTrait.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Traits;
+
+if (version_compare(phpversion('redis'), '6.0.2', '>')) {
+    /**
+     * @internal
+     */
+    trait RedisCluster6ProxyTrait
+    {
+        public function getex($key, $options = []): \RedisCluster|string|false
+        {
+        }
+
+        public function publish($channel, $message): \RedisCluster|bool|int
+        {
+            return ($this->lazyObjectState->realInstance ??= ($this->lazyObjectState->initializer)())->publish(...\func_get_args());
+        }
+
+        public function waitaof($key_or_address, $numlocal, $numreplicas, $timeout): \RedisCluster|array|false
+        {
```

Comment:
why aren't those calling the real instance ?

Reply:
copy and paste mistake